Face/FindSimilar/Program.cs

```diff
@@ -0,0 +1,258 @@
+        {
+            IList<Guid?> targetFaceIds = new List<Guid?>();
+            foreach (var targetImageFileName in TargetImageFileNames)
+            {
+                // Read image file.
+                using (FileStream stream = new FileStream(Path.Combine("Images", targetImageFileName), FileMode.Open))
+                {
+                    // Detect faces from image stream.
+                    IList<DetectedFace> detectedFaces = await Client.Face.DetectWithStreamAsync(stream);
+                    if (detectedFaces == null || detectedFaces.Count == 0)
+                    {
+                        Console.WriteLine($"[Error] No face detected from image `{targetImageFileName}`.");
+                        return;
+                    }
+
+                    Console.WriteLine($"{detectedFaces.Count} faces detected from image `{targetImageFileName}`.");
+                    if (detectedFaces[0].FaceId == null)
+                    {
+                        Console.WriteLine("[Error] Parameter `returnFaceId` of `DetectWithStreamAsync` must be set to `true` (by default) for verification purpose.");
+                        return;
+                    }
+
+                    // Add detected faceId to targetFaceIds.
+                    targetFaceIds.Add(detectedFaces[0].FaceId);
+                }
+            }
+
+            // Read image file.
+            using (FileStream stream = new FileStream(Path.Combine("Images", SourceImageFileName), FileMode.Open))
+            {
+                // Detect faces from image stream.
+                IList<DetectedFace> detectedFaces = await Client.Face.DetectWithStreamAsync(stream);
+                if (detectedFaces == null || detectedFaces.Count == 0)
+                {
+                    Console.WriteLine($"[Error] No face detected from image `{SourceImageFileName}`.");
+                    return;
+                }
+
+                Console.WriteLine($"{detectedFaces.Count} faces detected from image `{SourceImageFileName}`.");
+                if (detectedFaces[0].FaceId == null)
+                {
+                    Console.WriteLine("[Error] Parameter `returnFaceId` of `DetectWithStreamAsync` must be set to `true` (by default) for verification purpose.");
+                    return;
+                }
+
+                // Find similar example for find similar faces from targetFaceIds with similar confidence.
+                IList<SimilarFace> similarResults = await Client.Face.FindSimilarAsync(detectedFaces[0].FaceId.Value, null, null, targetFaceIds);
+                if (similarResults.Count == 0)
+                {
+                    Console.WriteLine($"Not found the same with `{detectedFaces[0].FaceId}` from similar result.{Environment.NewLine}");
+                }
+
+                foreach (var similarResult in similarResults)
+                {
+                    Console.WriteLine(
+                        similarResult.Confidence > 0.5
```

Comment:
Sorry my fault, all similar results returned are with confidence greater than 0.5. We can remove this. Same for all the three functions.

Reply:
done